src/components/Country/CountriesList.js

```diff
@@ -0,0 +1,48 @@
+import PropTypes from 'prop-types';
```

Comment:
- [ ] Please, consider that you have 1 test failed
<img width="587" alt="Screen Shot 2022-05-20 at 16 38 56" src="https://user-images.githubusercontent.com/49064106/169529819-a83e3e20-1c4a-453d-9fff-793c2bda737b.png">
<img width="545" alt="Screen Shot 2022-05-20 at 16 39 47" src="https://user-images.githubusercontent.com/49064106/169529964-f4b7341f-d6fb-44bc-a4e1-91eee2744ea9.png">

Reply:
Hi Elmar, I will look into it, see what might be causing the issue.